Fluent bit loki plugin (#5589)
kyma-project/kyma: docs/logging/03-03-configure-logcollector.md

> ---

Reviewer: can we change the name of the doc to `08-01-configure-logcollector.md` ? So that it's a tutorial :)

title: Log collector configuration examples

> type: Details

Reviewer: ```suggestion
type: Tutorials
```

---

> By default, Loki comes with the [promtail](https://github.com/grafana/loki) log collector configuration. Additionally, Kyma supports other log collectors, such as [Fluent Bit](https://fluentbit.io/) you can easily configure.

Reviewer: ```suggestion
By default, Loki comes with the [promtail](https://github.com/grafana/loki) log collector configuration. Additionally, Kyma supports other log collectors you can easily configure, such as [Fluent Bit](https://fluentbit.io/).
```

By default, Loki comes with the [promtail](https://github.com/grafana/loki) log collector configuration. Additionally, kyma supports other log collector, such as [fluent-bit](https://fluentbit.io/).

This is an example of Loki configuration using fluent-bit and elesticsearch:

File `values.yaml` from kyma logging helm chart have to be configured like following example:
```yaml
logcollector:

>   name: promtailfleunt-bit

Reviewer: Shouldn't this one be either `promtail` or `fluent-bit`? 🤔

> 1. Override the [`values.yaml`](https://github.com/kyma-project/kyma/blob/master/resources/logging/values.yaml) file includes the defined **logcollector** parameter. See the example: 

Reviewer: ```suggestion
1. Override the [`values.yaml`](https://github.com/kyma-project/kyma/blob/master/resources/logging/values.yaml) file to include the following **logcollector** parameter:
```

>   name: fleunt-bit

Reviewer: ```suggestion
  name: fluent-bit
```

For details on configurable parameters and overrides, see [this](/components/logging/#configuration-configuration) document.
2. Add the following configuration to the `fluent-bit-configmap.yaml` file for Fluent Bit to forward logs to Elasticsearch. 

> >**NOTE:** Configure **{FLUENT_ELASTICSEARCH_HOST}** and  **{FLUENT_ELASTICSEARCH_PORT}**  environment variables accordingly for your Elasticsearch deployment.

Reviewer: Let's move this note after the config, it looks better this way.
